pages_domains.go

```diff
@@ -144,8 +146,9 @@ func (s *PagesDomainsService) CreatePagesDomain(pid interface{}, opt *CreatePage
 // GitLab API docs:
 // https://docs.gitlab.com/ce/api/pages_domains.html#update-pages-domain
 type UpdatePagesDomainOptions struct {
-	Cerificate *string `url:"certifiate" json:"certifiate"`
-	Key        *string `url:"key" json:"key"`
+	Cerificate     *string `url:"certifiate" json:"certifiate"`
```

Comment:
I don't want to do it as a part of this PR (I'm worried about backwards compatibility), but I wonder if there's a good way to fix the missing `t` in `Certificate` here.

Reply:
Please do fix it in your PR. Since nobody reported this typo, I'm inclined to think that it's not being used much and it's clearly wrong.